packages/react-native-bpk-component-phone-input/src/BpkFlag.js

```diff
@@ -39,10 +41,18 @@ export type Props = {
   style: ViewStyleProp,
 };
 
-const styles = StyleSheet.create({
-  flag: {
-    borderColor: colorSkyGrayTint06,
-    borderWidth: 1, // eslint-disable-line backpack/use-tokens
+const styles = BpkDynamicStyleSheet.create({
```

Comment:
So for defining the light and dark mode colours the way it is done should be - which is why the tests have changed to show the style as `undefined`:

```suggestion
const styles = BpkDynamicStyleSheet.create({
   flag: {
     borderColor: {
      light: colorSkyGratTint06,
      dark: colorBlack
     },
     borderWidth: 1,
   }
});
```

Reply:
Ahh, my bad. Got it.